src/bloqade/ir/location/list.py

```diff
@@ -61,20 +61,24 @@ def __iter__(self):
 
 start = ListOfLocations()
 """
-    - Program starting node
-    - Possible Next <LevelCoupling>
-
-        -> `start.rydberg`
-            :: address rydberg level coupling
-
-        -> `start.hyperfine`
-            :: address hyperfine level coupling
-
-    - Possible Next <AtomArragement>
-
-        -> `start.add_locations(List[Tuple[int]])`
-            :: add multiple atoms to current register
-
-        -> `start.add_location(Tuple[int])`
-            :: add atom to current register
+A Program starting point, alias of empty
+[`ListOfLocations`][bloqade.ir.location.list.ListOfLocations].
+
+- Next possible steps to build your program are:
+- Specify which level coupling to address with:
+    - |_ `start.rydberg`: for [`Rydberg`][bloqade.builder.coupling.Rydberg]
```

Comment:
I feel asking people to type `|_` is a bit not scalable, what was the purpose of this again? to replace the arrows?

Reply:
Yes, this was purely an aesthetic choice. This is more a byproduct of when I first started writing these docstrings, I had everything as bullet points and realized I wanted a way to distinguish between "text" bullet points and "next options" bullet points so I ripped the style of the ASCII tree printer.

These can be removed altogether now that I've tried to enforce the idea of a standard paragraph of text for the intro/explanation and reserved bullet points for "next options". 

In an ideal world, I would think of no bullet points at all and something like this from `treelib` for `next options`: 
```
Harry
├── Bill
└── Jane
    ├── Diane
    │   └── Mary
    └── Mark
```
But it would make it more fragile for human editors.

Reply:
In principle we can auto-generate these guides by generating part of the docstring inside a decorator (like how `@wraps` work)